core/info/service.go

```diff
@@ -0,0 +1,66 @@
+package info
+
+import (
+	"context"
+	"time"
+)
+
+type BlockService interface {
+	GetBlockInfo(context.Context) BlockInfo
+}
+
+type BlockInfo interface {
+	GetHeight() int64      // GetHeight returns the height of the block
+	GetHeaderHash() []byte // GetHeaderHash returns the hash of the block header
+	GetTime() time.Time    // GetTime returns the time of the block
+	GetChainID() string    // GetChainId returns the chain ID of the block
+}
+
+type CometService interface {
+	GetCometInfo(context.Context) CometInfo
+}
+
+type CometInfo interface {
+	GetMisbehavior() []Misbehavior // Misbehavior returns the misbehavior of the block
```

Comment:
How about moving the comet stuff to a separate package `core/comet`. Do we think this will stay stable between comet releases?